src/main/java/jalau/at18/azul/AzulFrame.java

```diff
@@ -0,0 +1,32 @@
+package jalau.at18.azul;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class AzulFrame extends JFrame {
+    private JPanel player1;
+    private JPanel player2;
+    private JPanel factory;
+    private static final int DEFAULT_POS = 305;
+    private static final int DEFAULT_POS1 = 500;
+    private static final int DEFAULT_POS2 = 10;
+    private static final int DEFAULT_SIZE = 300;
+    private static final int DEFAULT_SIZE1 = 610;
+    public AzulFrame() {
+        player1 = new JPanel();
+        this.add(player1);
+        player1.setBounds(0, 0, DEFAULT_POS1, DEFAULT_SIZE);
+        player1.setLayout(null);
+        player1.setBorder(BorderFactory.createLineBorder(Color.red));
```

Comment:
I think we can create a 
```java
class PlayerPanel extends JPanel {}
```
here, so when you add them to your `AzulFrame`, you simply do
```java
add(new PlayerPanel()); // PanelPlayer constructor can include parameters for things that change for each instance
add(new PlayerPanel());
add(new PlayerPanel());
```

Reply:
Already create the class PlayerPanel and now the AzulFrame class only call the playerpanel class to add to the frame.
Also create the unit test for the playerpanel